Lib/test/test_coroutines.py

```diff
@@ -2414,7 +2413,6 @@ async def corofn():
                 del coro
                 support.gc_collect()
 
-                self.assertEqual(repr(cm.unraisable.object), coro_repr)
```

Comment:
We should test the formatted message instead of the object attribute.

Reply:
Right, I added checks on *err_msg*.